internal/webhooks/clusterclass.go

```diff
@@ -176,6 +179,10 @@ func (webhook *ClusterClass) validate(ctx context.Context, oldClusterClass, newC
 		// Ensure no MachineHealthCheck currently in use has been removed from the ClusterClass.
 		allErrs = append(allErrs,
 			validateUpdatesToMachineHealthCheckClasses(clusters, oldClusterClass, newClusterClass)...)
+
+		// Ensure Topology metadata in every clusters is valid.
+		allErrs = append(allErrs,
+			validateClustersTopologyMetadata(clusters)...)
```

Comment:
```suggestion
```
I wouldn't add this here. I think it's enough if we validate clusters in the cluster webhook

Reply:
Agree. removed this part